Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraint.cs

```diff
@@ -525,6 +547,19 @@ private static bool IsPalmFacingCamera(IMixedRealityController hand)
             return false;
         }
 
+        private static MixedRealityPose? GetPalmPose(IMixedRealityController hand)
+        {
+            MixedRealityPose palmPose;
+            var jointedHand = hand as IMixedRealityHand;
+
+            if ((jointedHand != null) && jointedHand.TryGetJoint(TrackedHandJoint.Palm, out palmPose))
```

Comment:
There's some copy pasta now here between this and the above function:

IsPalmFacingCamera

This is a nit but they're fairly close, would be good stewardship to consolidate (i.e. IsPalmFacingCamera calls -> GetPalmPose)

Reply:
Ah, good point.